grafana/dashboards/cmode/smb2.json

```diff
@@ -86,6 +86,36 @@
         "x": 0,
         "y": 0
       },
+      "id": 94,
+      "panels": [],
+      "title": "Important Information",
+      "type": "row"
+    },
+    {
+      "datasource": "${DS_PROMETHEUS}",
+      "gridPos": {
+        "h": 2,
+        "w": 24,
+        "x": 0,
+        "y": 1
+      },
+      "id": 92,
+      "options": {
+        "content": "This dashboard requires ZapiPerf collector. It is currently unavailable for RestPerf Collector. Refer https://github.com/NetApp/harvest/issues/1756",
```

Comment:
maybe this?

This dashboard requires the ZapiPerf collector. SMB2 is currently unavailable via REST. Refer https://github.com/NetApp/harvest/issues/1756